blueprint/index.md

```diff
@@ -0,0 +1,169 @@
+```
+
+### Create an Implicit Grant OAuth
+
+1. Login to your Genesys Cloud organization and create a new OAuth Credential (Implicit Grant). [Create an OAuth Client](https://help.mypurecloud.com/articles/create-an-oauth-client/)
+2. Add **https://localhost/** to the **Authorized redirect URIs**. Note: If you've changed the **redirecUri** value in the config file, then you need to add that new URI instead.
+3. Add the following in the Scopes section:
+    * analytics
+    * authorization
+    * presence
+    * routing
+    * users
+4. Save the Client ID for use in the configuration of the project.
+
+### Create a Client Credentials OAuth Grant for Genesys Cloud
+
+1. Login to your Genesys Cloud organization and create a new OAuth Credential (Client Credentials Grant). [Create an OAuth Client](https://help.mypurecloud.com/articles/create-an-oauth-client/)
+2. Select all of the admin **roles** for the OAuth client.
+3. Take note of the Client ID and Client Secret.
+
+### Update Configuration File
+
+Modify the values in the configuration file before running the app. Use the values from the OAuth Client your created in the last step as follows:
+
+clientConfig.js:
+
+```javascript
+export const clientConfig = {
+  GENESYS_CLOUD_CLIENT_ID: '<YOUR CLIENT ID HERE>',
+  REDIRECT_URI: '<YOUR PRODUCTION URI HERE>',
+};
+```
+
+### Run the App
+
+Open a terminal and set the working directory to the root directory of the project, then run the following:
+
+```bash
+npm run install
+npm run start
+```
+
+## Sample App Overview
+
+### Genesys Cloud Utils
+
+The `genesysCloudUtils` file contains the intermediate functions that in turn call Genesys Cloud SDK methods. These functions return promises that are handled upon resolution in the file and in the invoking components themselves.
+
+### User Home Page
+
+The home page serves as a profile page for the logged in user.  It formats meaningful information about the user into sections on the page.  Profile image, name, and user presence are at the top, and more detailed information about the user is found at the bottom.
+
+### User Search
+
+This search page allows the user to search listings of other users by name or email.  The results are cards that act as miniature profile pages for the found users.
+
+### User Queues
+
+The queue page contains listings for all the queues the logged in user belongs to.  The queue name is displayed along with some statistics about the queue. An example of admin functionality is also found here in the form of a button that logs out all the members of a queue.
+
+## Configuring the React Project to use Genesys Cloud SDK
+
+Now we'll look at the steps needed to integrate the Genesys Cloud SDK into your own React app.
+
+### Creating a React Project
+
+If you are creating a new app from scratch, run the following command in a terminal in the directory of your choice:
+
+```bash
+npx create-react-app name-of-your-app --template typescript
```

Comment:
list and link to npx and create react app in the resources; those are things people are going to have to install